pandas/core/frame.py

```diff
@@ -7835,7 +7837,7 @@ def f(x):
             return op(x, axis=axis, skipna=skipna, **kwds)
 
         def _get_data(axis_matters):
-            if filter_type is None or filter_type == "numeric":
```

Comment:
would it be clearer to have "numeric" as the default value of filter_type instead of None. Again maybe a docstring would help.